File: TransCoder_tokenized_test_set_functions/cpp/MINIMUM_PERIMETER_N_BLOCKS.cpp
int minPerimeter ( int n ) {
  int l = sqrt ( n );
  int sq = l * l;
  if ( sq == n ) return l * 4;
  else {
    long long int row = n / l;
    long long int perimeter = 2 * ( l + row );
    if ( n % l != 0 ) perimeter += 2;
    return perimeter;
  }
}

File: results/java_codex/MINIMUM_PERIMETER_N_BLOCKS.java
long minPerimeter(int n) {
        int l = (int) sqrt(n);
        int sq = l * l;
        if (sq == n) return l * 4;
        else {
            long row = n / l;
            long perimeter = 2 * (l + row);
            if (n % l != 0) perimeter += 2;
            return perimeter;
        }
    }